rust/agama-lib/share/profile.schema.json

```diff
@@ -224,54 +219,314 @@
+                          }
+                        }
+                      ]
+                    }
+                  }
+                }
+              },
+              "else": {
+                "required": ["policy"],
+                "properties": {
+                  "actions": {
+                    "type": "array",
+                    "maxItems": 0
+                  }
+                }
+              }
+            },
+            "volumes": {
+              "title": "Set of volumes (file systems) to create",
+              "type": "array",
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "required": ["mount"],
+                "properties": {
+                  "mount": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": ["path"],
+                    "properties": {
+                      "path": {
+                        "title": "Mount path",
+                        "type": "string"
+                      },
+                      "options": {
+                        "title": "Options to add to the fourth field of fstab",
+                        "type": "array",
+                        "items": { "type": "string" }
+                      }
+                    }
+                  },
+                  "filesystem": {
+                    "title": "File system of the volume",
+                    "oneOf": [
+                      {
+                        "description": "File system type",
+                        "enum": [
+                          "bcachefs", "btrfs", "exfat", "ext2", "ext3", "ext4", "f2fs", "jfs",
+                          "nfs", "nilfs2", "ntfs", "reiserfs", "swap", "tmpfs", "vfat", "xfs"
+                        ]
+                      },
+                      {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "properties": {
+                          "btrfs": {
+                            "title": "Specification of a Btrfs file system",
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                              "snapshots": {
+                                "title": "Whether Btrfs snapshots should be configured",
+                                "type": "boolean"
+                              }
+                            }
+                          }
+                        },
+                        "required": ["btrfs"]
+                      }
+                    ]
+                  },
+                  "size": {
+                    "title": "Size limits",
+                    "oneOf": [
+                      { "const": "auto" },
+                      { "$ref": "#/$defs/sizeString" },
+                      { "$ref": "#/$defs/sizeInteger" },
```

Comment:
We could replace these 2 lines with `sizeValue`, right?

Reply:
Yes. Thanks!